contrib/ci/drone-static-upload.sh

```diff
@@ -50,9 +50,9 @@ elif [ -e build-docs ]; then
     archive="$base.tar.xz"
     cp -av build-docs/docs/mkdocs.yml build-docs/docs/markdown "$base"
     tar cJvf "$archive" "$base"
-elif [ -e build-mac ]; then
+elif [ -e build/macos ]; then
```

Comment:
Please don't.  I routinely use multiple `build*` directories, including `build` for manual builds.  Nesting it under `build/` is a nuisance in multiple ways.